cache/src/main/java/net/runelite/cache/ItemManager.java

```diff
@@ -110,6 +116,54 @@ public void java(File java) throws IOException
 		}
 	}
 
+	public void variations(File out) throws IOException
```

Comment:
I don't think this belongs in here.

Reply:
This depends on https://github.com/runelite/runelite/pull/6311

so whatever is done there will be reflected here

Reply:
I already removed that commit from that PR, so you can just update